File: twiml/voice/recording/recording-1/recording-1.5.x.rb
require 'twilio-ruby'

response = Twilio::TwiML::VoiceResponse.new
response.start do |start|
    start.recording(channels: 'dual', recording_status_callback: 'https://example.com/your-callback-url')
end
response.say(message: 'The recording has started.')

puts response

File: twiml/voice/recording/recording-1/recording-1.5.x.cs
using System;
using Twilio.TwiML;
using Twilio.TwiML.Voice;


class Example
{
    static void Main()
    {
        var response = new VoiceResponse();
        var start = new Start();
        start.Recording(channels: "dual",
            recordingStatusCallback: "https://example.com/your-callback-url");
        response.Append(start);
        response.Say("The recording has started.");

        Console.WriteLine(response.ToString());
    }
}
